instead off adding mdbook-linkcheck, skip it since there isn't much use for us
diff --git a/doc/manual/book.toml b/doc/manual/book.toml
index 73fb7e7..455e1b5 100644
--- a/doc/manual/book.toml
+++ b/doc/manual/book.toml
@@ -14,6 +14,7 @@ command = "jq --from-file doc/manual/anchors.jq"
 [output.linkcheck]
 # no Internet during the build (in the sandbox)
 follow-web-links = false
+optional = true
 
 # mdbook-linkcheck does not understand [foo]{#bar} style links, resulting in
 # excessive "Potential incomplete link" warnings. No other kind of warning was
